### PR TITLE
Added details about installing addons package

### PR DESCRIPTION
**Pull request by awanlin**
Signed-off-by: Andre Wanlin <67169551+awanlin@users.noreply.github.com>

## Hey, I just made a Pull Request!

Added details about installing `@backstage/plugin-techdocs-module-addons-contrib` package

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by awanlin** [comment]
@emmaindal finally got around to making a PR for this based on our Discord chat: https://discord.com/channels/687207715902193673/979390017334026260

File: docs/features/techdocs/addons.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ Addons are rendered in the order in which they are registered.
 
 ## Installing and using Addons
 
+To start using Addons you need to add the `@backstage/plugin-techdocs-module-addons-contrib` package you your app, you can do that by running this command form the root of your project: `yarn add --cwd packages/app @backstage/plugin-techdocs-module-addons-contrib`
```
Comment (adamdmharvey) [inline docs/features/techdocs/addons.md:55]:
```suggestion
To start using Addons you need to add the `@backstage/plugin-techdocs-module-addons-contrib` package to your app. You can do that by running this command from the root of your project: `yarn add --cwd packages/app @backstage/plugin-techdocs-module-addons-contrib`
```

Minor typo, might suggest this?

**Review by adamdmharvey — CHANGES_REQUESTED** [review]

File: docs/features/techdocs/addons.md (hunk 1 above)
Comment (camilaibs) [reply docs/features/techdocs/addons.md:55]: Hi @awanlin thanks again for contributing, could you consider this suggestion?

Comment (awanlin) [reply docs/features/techdocs/addons.md:55]: Yes, for sure, clearly my wiring skills failed me at the end of the day

Comment (awanlin) [reply docs/features/techdocs/addons.md]: Ok, all cleaned up

**Comment by awanlin** [comment]
@camilaibs and @adamdmharvey, one other thing we ran into when setting up Addons was that we have a customer reader page. I wasn't sure if they were supported with that use case but they are. Should we update this documentation with something about that? Maybe just a code example for that case? I could add that to this PR

**Review by adamdmharvey — APPROVED** [review]

**Comment by camilaibs** [comment]
> @camilaibs and @adamdmharvey, one other thing we ran into when setting up Addons was that we have a customer reader page. I wasn't sure if they were supported with that use case but they are. Should we update this documentation with something about that? Maybe just a code example for that case? I could add that to this PR

This is an excellent idea, thank you for being so generous in making this contribution further!

**Review by camilaibs — APPROVED** [review]

**Pull request merged**